packages/core/src/extension.ts

```diff
@@ -194,6 +195,9 @@ export function registerGenericCommands(extensionContext: vscode.ExtensionContex
         }),
         Commands.register(`aws.${contextPrefix}.aboutExtension`, async () => {
             await aboutExtension()
+        }),
+        Commands.register(`aws.${contextPrefix}.fetchFeatureConfigs`, async () => {
+            await FeatureConfigProvider.instance.fetchFeatureConfigs()
```

Comment:
why is this a command instead of a plain old function? I don't see any cases in the codebase that require it to be a command. Is this intended for users to invoke from the cmd palette?

Reply:
The intention is to allow customer to do `await vscode.commands.executeCommand('aws.amazonq.fetchFeatureConfigs')` or `await vscode.commands.executeCommand('aws.toolkit.fetchFeatureConfigs')`. 

May be we could just make it a function instead.